appveyor.yml

```diff
@@ -1,4 +1,6 @@
 image: Visual Studio 2017
+install:
+  - set PATH=C:\Ruby25-x64\bin;%PATH%
```

Comment:
Is this always installed in the same location on Windows?

Reply:
This is the configuration for `appveyor` so it will always be installed here, unless they change it... https://www.appveyor.com/docs/lang/ruby/#installation